config.js

```diff
@@ -346,12 +346,12 @@ const config = {
       },
       {
         text: 'About the docs ',
-        url: 'about/about-the-docs',
+        url: 'about/prisma-docs/about-the-docs',
```

Comment:
Did those not work? Shouldn't they be covered by redirects?

Reply:
Maybe the redirect doesnt cover the homepage? 🤷‍♂️

Reply:
It is in the redirects file, just checked

Reply:
FWIW these links are defined in a config file at the projects root. Maybe the redirects only search in the /content folder

Reply:
I remember there were two locations to define redirects in docs, for external links (via homepage_v8) and internal ones (some json file probably). Could that be it, that they are only defined externally?

Reply:
Yep, that is what I recalled probably.

Reply:
Ive been setting them on the homepage_v8 repo. I didnt even know this array existed. Im not even sure its used. @nilubava would know i guess.

But to the original point. That link now works 😄